pkg/wordmodifiers/filters.py

```diff
@@ -0,0 +1,60 @@
+import os
+import re
+
+
+class CaseFolder:
```

Comment:
I moved out the normalizer/filtering methods to their own module since we end up reusing them in other areas of the program